_posts/en/newsletters/2024-06-14-newsletter.md

```diff
@@ -0,0 +1,149 @@
+  chainstate means these indexes will be considered out-of-sync. As per
+  the previous question, that means they will first perform a background
+  sync before switching to the validation interface. With this PR, the
+  optional indexes remain in sync with the chainstate, and as such no
+  background sync is required. Note: background sync only starts after
+  reindex has completed, whereas processing validation events happens in
+  parallel."
+  a2link="https://bitcoincore.reviews/30132#l-70"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 24.05][] is a release of the next major
+  version of this popular LN node implementation.  FIXME:harding add
+  details
+
+- [Bitcoin Core 27.1rc1][] is a release of a maintenance
+  version of the predominant full node implementation. FIXME:harding add
+  details
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+FIXME:Gustavojfe
+
+- [Bitcoin Core #29496][] policy: bump TX_MAX_STANDARD_VERSION to 3
+
+- [Bitcoin Core #28307][] rpc, wallet: fix incorrect segwit redeem script size limit
+
+- [Bitcoin Core #30047][] refactor: Model the bech32 charlimit as an Enum ; mainly just looking for a quick mention that this is being done for silent payments -harding
+
+- [Bitcoin Core #28979][] wallet, rpc: document and update `sendall` behavior around unconfirmed inputs
+
+- [Eclair #2854][] and [LDK #3083][] both related to [BOLTs #1163][]
+
+- [LND #8491][] invoice_cltv_expiry
+
+- [LDK #3080][] Optional compact blinded path creation
+
+- [LDK #3072][] Reintroduce addresses to NodeAnnouncementInfo. ; this
+  might not be notable - fell free to delete if your research indicates
+  it's just something minor -harding
+
+- [BIPs #1551][] Add BIP 353: DNS Payment Instructions
+
+{% assign four_days_after_posting = page.date | date: "%s" | plus: 345600 | date: "%Y-%m-%d 14:30" %}
+{% include snippets/recap-ad.md when=four_days_after_posting %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="29496,28307,30047,28979,2854,3083,1163,8491,3080,3072,1551,30132" %}
+[beast post]: https://delvingbitcoin.org/t/proposing-a-p2qrh-bip-towards-a-quantum-resistant-soft-fork/956
+[quantum draft]: https://github.com/cryptoquick/bips/blob/61aac1a91a596cdeed75ed7f5c5c19b0b4b923ef/bip-p2qrh.mediawiki
```

Comment:
The draft has gone through several additional iterations since it was first posted. Here's the latest copy.

```suggestion
[quantum draft]: https://github.com/cryptoquick/bips/blob/e1fa27cb25deee8d58bdb9a64c112dfdf09e216d/bip-p2qrh.mediawiki
```